src/checkbox/README.md

```diff
@@ -0,0 +1,200 @@
+# Checkbox Component
+
```

Comment:
1-liner: `A checkbox control that is used for selecting multiple options from a set.`